webpack.common.config.js

```diff
@@ -121,7 +121,8 @@ module.exports = function (outputFilename) {
           exclude: /node_modules/,
           loader: 'babel-loader',
           query: {
-            presets: ['env']
+            presets: ['env'],
+            plugins: ['transform-runtime']
```

Comment:
Am curious - if we have transform-runtime + babel-polyfill (for the cdn version), does that mean we have two copies of all the polyfills? In general, what is the difference in file size in the built file between this commit and current master?

Reply:
Before
okta-sign-in.min.js - 989 kB
okta-sign-in-no-jquery.js- 891 kB
okta-sign-in.entry.js - 1815 kB

After
okta-sign-in.min.js - 1020 kB
okta-sign-in-no-jquery.js- 921 kB
okta-sign-in.entry.js - 1640 kB

Latest commit
okta-sign-in.min.js - 989 kB
okta-sign-in-no-jquery.js- 891 kB
okta-sign-in.entry.js - 1640 kB